index.html

```diff
@@ -9,57 +9,246 @@
+                    </div>
+                    <div class="btn-container">
+                        <div class="btn"><a href="#">See project</a></div>
+                    </div>
+                </div>
+            </section>
+        </section>
+
+        <!-- About Myself -->
+
+        <section class="aboutme">
+            <section class="aboutme-header">
+                <h1>About Myself</h1>
                 <p>
-                    I’m a software developer! I can help you build a product , feature
-                    or website Look through some of my work and experience! If you
-                    like what you see and have a project you need coded, don’t
-                    hestiate to contact me.
+                    Hello I’m a software developer! I can help you build a 
+                    product, feature or website. Look through some of my 
+                    work and experience! If you like what you see and have 
+                    a project you need coded, don’t hesitate to contact me.
                 </p>
-                <h3>LET’S CONNECT</h3>
-                <ul class="social-icons">
-                    <li>
-                        <a href="#"><img src="./assets/social/twitter.svg" alt="Twitter"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/Linkedin.svg" alt="Linkedin"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
-                    </li>
-                </ul>
-            </div>
+                <div class="letcon">
+                    <h3 class="let_connect">LET’S CONNECT</h3>
+                    <ul class="social-icons">
+                        <li>
+                            <a href="#"><img src="./assets/social/twitter.svg" alt="Twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/Linkedin.svg" alt="Linkedin"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="btn"><a href="#">Get resume</a></div>
```

Comment:
- [ ] Kindly complete the text inside the div to match what is on the Figma design

### Your Design
![your-btn](https://user-images.githubusercontent.com/26597432/236418790-953aab00-ecf0-475b-965d-1278862c9e5a.png)

### Figma Design
![temp-btn](https://user-images.githubusercontent.com/26597432/236418839-84cd8daf-5c56-40e8-a333-37d5d97871d0.png)